src/pages/EditReportFieldPage.tsx

```diff
@@ -0,0 +1,114 @@
+import React, {useEffect} from 'react';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
+import FullPageNotFoundView from '@components/BlockingViews/FullPageNotFoundView';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Navigation from '@libs/Navigation/Navigation';
+import * as ReportActions from '@src/libs/actions/Report';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {PolicyReportFields, Report} from '@src/types/onyx';
+import EditReportFieldDatePage from './EditReportFieldDatePage';
+import EditReportFieldDropdownPage from './EditReportFieldDropdownPage';
+import EditReportFieldTextPage from './EditReportFieldTextPage';
+
+type EditReportFieldPageOnyxProps = {
+    /** The report object for the expense report */
+    report: OnyxEntry<Report>;
+
+    /** Policy report fields */
+    policyReportFields: OnyxEntry<PolicyReportFields>;
+};
+
+type EditReportFieldPageProps = EditReportFieldPageOnyxProps & {
+    /** Route from navigation */
+    route: {
+        /** Params from the route */
+        params: {
+            /** Which field we are editing */
+            fieldID: string;
+
+            /** reportID for the expense report */
+            reportID: string;
+
+            /** policyID for the expense report */
+            policyID: string;
+        };
+    };
+};
+
+function EditReportFieldPage({route, report, policyReportFields}: EditReportFieldPageProps) {
+    const policyReportField = policyReportFields?.[route.params.fieldID];
+    const reportFieldValue = report?.reportFields?.[policyReportField?.fieldID ?? ''];
+
+    // Decides whether to allow or disallow editing a money request
+    useEffect(() => {}, []);
```

Comment:
Maybe tidy this away for now?

Reply:
Doesn't really matter. It would be filled in an upcoming PR anyways. I think this would be a reminder to not miss this case!